package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -449,12 +445,15 @@
 """
 from __future__ import print_function, division, absolute_import
 
-import warnings
-
 from six.moves import range, zip_longest
-
+import logging
+import warnings
 import numpy as np
 import MDAnalysis.analysis.hbonds
+from MDAnalysis.lib.log import ProgressMeter
```

Comment:
ProgressMeter should not be here. It's been deprecated and removed everywhere as part of PR #2617 .

Please remove.